### ISSUE TITLE
ReadyToRun does not work on Unix

### ISSUE THREAD

**Issue by jkotas** [dotnet/coreclr#2710]
Run: `crossgen -readytorun foo.dll`
Result: `Invalid parameter -readytorun`

It was supposed to be enabled by #1022, but it did not seem to actually ever work.

We should also check why it was not caught by the test (https://github.com/dotnet/coreclr/tree/master/tests/src/readytorun).

**Comment by janvorli** [comment]
@jkotas This test was not enabled to run on Unix yet. It belongs to a bunch of tests that use CLRTestBatchPreCommands and those don't work on Unix since there is no support for Unix shell script generation. 
As for why the Linux crossgen doesn't support the option, there is a `FEATURE_CROSSGEN_COMPILER` that we have enabled for windows only. When I was enabling the crossgen on Linux, I have enabled only the `FEATURE_CROSSGEN` and I have missed the fact that there is the other one too.
I'll fix that.
